.github/workflows/tests.yml

```diff
@@ -64,8 +64,20 @@ jobs:
           --no-watch-fs \
           -Drobolectric.enabledSdks=${{ matrix.api-versions }} \
           -Drobolectric.alwaysIncludeVariantMarkersInTestName=true \
+          -Dorg.gradle.workers.max=2 \
+          -x :integration_tests:nativegraphics:test
```

Comment:
@hoisie Is it for test purpose only?

Reply:
It's complicated... I was seeing weird CI failures when running native graphics tests in the regular tests CI workflow. If I separate the graphics tests so they are in a separate process it fixes the issue.

I am thinking of running the graphics tests on our M1 mini runner for now.

We may also upgrade to a paid plan to use larger CI machines with more than 7 GiB ram.

Reply:
Oh. Looks like all tests consume too much RAM. It's great to separate native graphics tests.

> We may also upgrade to a paid plan to use larger CI machines with more than 7 GiB ram.

Is it necessary? At least current CI time is accepted after nativeruntime building removed. We can trade time for space now.